ui/packages/atlasmap/src/Views/ColumnMapperView/Links/SourceTargetLinks.tsx

```diff
@@ -35,11 +35,12 @@ export const SourceTargetLinks: FunctionComponent<ISourceTargetLinksProps> = ({
 
   return (
     <>
-      {sortedMappings.map((m) => (
+      {sortedMappings.map((m, index) => (
         <MappingLines
           mapping={m}
           onClick={() => onSelectMapping(m)}
           isSelected={m.id === selectedMappingId}
+          key={index}
```

Comment:
It's always better to give a meaningful key if possible to help React with the reconciliation. In this case, `m.id` could be a better option.

Reply:
My understanding was just that they need to be unique. I'll go ahead and make the change, but just for my own understanding, can you expound on how and why the id's would be better?

Reply:
In short, a key should be stable and not change between renders. With an index in this case if a mapping gets added/remove keys will be different for some elements making React's job harder. For more details check the doc https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
k, that makes sense. I've updated the code with this in mind.